packages/core/src/components/fieldset/fieldset.scss

```diff
@@ -0,0 +1,7 @@
+@mixin Fieldset() {
+  .ods-fieldset {
+    border: unset;
```

Comment:
💅🏻  [nitpick]

Some browsers have issues with the `unset` value.
Probably none that we care about, but still.

```suggestion
    border: 0;
```

or

```suggestion
    border: none;
```

Reply:
Linting does not allow "none" value for border:

<img width="1059" alt="Screenshot 2021-02-19 at 08 40 51" src="https://user-images.githubusercontent.com/20243687/108479891-2ebd1c00-728e-11eb-9dd1-919b2ff01715.png">

Going with "0".